src/metabase/email/messages.clj

```diff
@@ -226,3 +226,121 @@
+                                               (render/render-pulse-section timezone result))))))
+        message-ctx  (default-alert-context alert (alert-results-condition-text goal-value))]
+    (render-message-body "metabase/email/alert"
+                         (assoc message-ctx :pulse body :firstRunOnly? alert_first_only)
+                         (seq @images))))
+
+(def ^:private alert-condition-text
+  {:meets "when this question meets its goal"
+   :below "when this question goes below its goal"
+   :rows  "whenever this question has any results"})
+
+(defn- send-email! [user subject template-path template-context]
+  (email/send-message!
+    :recipients   [(:email user)]
+    :message-type :html
+    :subject      subject
+    :message      (stencil/render-file template-path template-context)))
+
+(defn- template-path [template-name]
+  (str "metabase/email/" template-name ".mustache"))
+
+;; Paths to the templates for all of the alerts emails
+(def ^:private new-alert-template (template-path "alert_new_confirmation"))
+(def ^:private you-unsubscribed-template (template-path "alert_unsubscribed"))
+(def ^:private admin-unsubscribed-template (template-path "alert_admin_unsubscribed_you"))
+(def ^:private added-template (template-path "alert_you_were_added"))
+(def ^:private stopped-template (template-path "alert_stopped_working"))
+(def ^:private deleted-template (template-path "alert_was_deleted"))
+
+(defn send-new-alert-email!
+  "Send out the initial 'new alert' email to the `CREATOR` of the alert"
+  [{:keys [creator] :as alert}]
+  (send-email! creator "You setup an alert" new-alert-template
+               (default-alert-context alert alert-condition-text)))
+
+(defn send-you-unsubscribed-alert-email!
+  "Send an email to `WHO-UNSUBSCRIBED` letting them know they've unsubscribed themselves from `ALERT`"
+  [alert who-unsubscribed]
+  (send-email! who-unsubscribed "You unsubscribed from an alert" you-unsubscribed-template
+               (default-alert-context alert)))
+
+(defn send-admin-unsubscribed-alert-email!
+  "Send an email to `USER-ADDED` letting them know `ADMIN` has unsubscribed them from `ALERT`"
+  [alert user-added {:keys [first_name last_name] :as admin}]
+  (let [admin-name (format "%s %s" first_name last_name)]
+    (send-email! user-added "You’ve been unsubscribed from an alert" admin-unsubscribed-template
+                 (assoc (default-alert-context alert) :adminName admin-name))))
+
+(defn send-you-were-added-alert-email!
+  "Send an email to `USER-ADDED` letting them know `ADMIN-ADDER` has added them to `ALERT`"
+  [alert user-added {:keys [first_name last_name] :as admin-adder}]
+  (let [subject (format "%s %s added you to an alert" first_name last_name)]
+    (send-email! user-added subject added-template (default-alert-context alert alert-condition-text))))
+
+(def ^:private not-working-subject "One of your alerts has stopped working")
```

Comment:
I think basically this whole namespace needs to be i18n'ed

Reply:
I didn't i18n this namespace intentionally. These strings are either email subjects or are snippets of text that are being injected into the emails. We don't currently have an i18n setup for these email templates, that I'm aware of. I was thinking if we i18n this, and had a translation, we would be in a situation where we have a Spanish email subject with a mostly English email body, but with a few Spanish phrases like 'results for you to see' thrown in. That seemed worse that an all English email.

I was thinking we'd need to address i18n for emails as a whole before i18n'ing this.